kubetest2/pkg/exec/doc.go

```diff
@@ -0,0 +1,20 @@
+/*
+Copyright 2019 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Package exec contains an interface for executing commands, along with helpers
+// TODO(bentheelder): add standardized timeout functionality & a default timeout
```

Comment:
this TODO is from upstream, if / when this is resolved I'll add the functionality to both

Reply:
In general this package is the same as upstream, but with all logging stripped out and some other unnecessary helpers dropped

Reply:
it's nice to see what command failed and even see a minimal stack trace in debug mode.

Reply:
right, but logging is not decided yet, I want to flesh out the interfaces first and then look at the state of klog